frontend/src/models/user.ts

```diff
@@ -66,6 +58,42 @@ export class User {
     }
 }
 
+export const logout =  async () => {
+    if (!ENV.IS_DEV_MODE) return
+    await fetch(`${ENV.API_ADDRESS}/development/users/log_out`, {
+        method: 'DELETE', credentials: 'include',
+    })
+}
+
+export const loginAsUser = async (id: string) => {
+    await fetch(`${ENV.API_ADDRESS}/development/users/${id}/log_in`, {
+        method: 'PUT', credentials: 'include',
+    })
+}
```

Comment:
this should be using `window.location = <href>` not fetch, OR change the caller to just use an `<a href={`${ENV.API_ADDRESS}/development/users/${id}/log_in`}>Become User</a>` tag.

Reply:
oh no sorry! I got confused and thought this was the impersonate call 😭   Sorry to confuse ya, this is fine since it's dev-only, although maybe someday we should update it as well